dev/build_release.py

```diff
@@ -199,6 +199,9 @@ def start_deb_build(self, name):
       target = 'buildDeb'
       extra_args = []
 
+    if self.__options.debug_gradle:
+      extra_args.append('--debug')
```

Comment:
I'm not sure how `--debug` interacts with `--stacktrace`, which is already passed. Just noting, we'll see how it goes.